DQM/L1TMonitorClient/python/L1TStage2EmulatorEventInfoClient_cfi.py

```diff
@@ -0,0 +1,350 @@
+                                QualityTestSummaryEnabled = cms.uint32(1)
+                                ),
+                            cms.PSet(
+                                QualityTestName = cms.string("InterMuonsDE_MismatchRatioMax0p05"),
+                                QualityTestHist = cms.string("L1TEMU/L1TdeStage2uGMT/intermediate_muons/OMTF_pos/data_vs_emulator_comparison/mismatchRatio"),
+                                QualityTestSummaryEnabled = cms.uint32(1)
+                                ),
+                            cms.PSet(
+                                QualityTestName = cms.string("InterMuonsDE_MismatchRatioMax0p05"),
+                                QualityTestHist = cms.string("L1TEMU/L1TdeStage2uGMT/intermediate_muons/OMTF_neg/data_vs_emulator_comparison/mismatchRatio"),
+                                QualityTestSummaryEnabled = cms.uint32(1)
+                                ),
+                            cms.PSet(
+                                QualityTestName = cms.string("InterMuonsDE_MismatchRatioMax0p05"),
+                                QualityTestHist = cms.string("L1TEMU/L1TdeStage2uGMT/intermediate_muons/EMTF_pos/data_vs_emulator_comparison/mismatchRatio"),
+                                QualityTestSummaryEnabled = cms.uint32(1)
+                                ),
+                            cms.PSet(
+                                QualityTestName = cms.string("InterMuonsDE_MismatchRatioMax0p05"),
+                                QualityTestHist = cms.string("L1TEMU/L1TdeStage2uGMT/intermediate_muons/EMTF_neg/data_vs_emulator_comparison/mismatchRatio"),
+                                QualityTestSummaryEnabled = cms.uint32(1)
+                                ),
+                            )
+                        ),
+                    cms.PSet(
+                        SystemLabel = cms.string("uGT"),
+                        HwValLabel = cms.string("Stage2uGT"),
+                        SystemDisable  = cms.uint32(0),
+                        QualityTests = cms.VPSet(
+                            cms.PSet(
+                                QualityTestName = cms.string(""),
+                                QualityTestHist = cms.string(""),
+                                QualityTestSummaryEnabled = cms.uint32(0)
+                                ),
+                            )
+                        ),
+                    ),
+
+    #
+    # for each L1 trigger object, give:
+    #     - ObjectLabel:  object label as used in enum L1GtObject
+    #     - ObjectDisable: emulator mask: if 1, the system is masked in the summary plot
+    #
+    # the position in the parameter set gives, in reverse order, the position in the reportSummaryMap
+    # in the trigger object column (right column)
+    L1Objects = cms.VPSet(
+                    cms.PSet(
+                        ObjectLabel = cms.string("TechTrig"),
```

Comment:
please try to avoid the duplicated 100s of lines below. Perhaps a loop?
@stahlleiton